CHANGELOG.md

```diff
@@ -11,12 +11,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
    Unreleased section, uncommenting the header as necessary.
 -->
 
-## Unreleased
+<!--## Unreleased-->
 
+## [2.0.0-alpha.35] - 2017-04-05
+
+* [minor breaking change] By default queries for features and warnings now traverse lazy imports. Added a query option to limit results only to those reachable by normal (strict) imports.
```

Comment:
FYI, I'd like to be consistent in our references to lazy vs normal imports. So far I've been using lazy vs eager. "strict" is a new term so far for that.